versioned_docs/version-25.10/releases/centreon-os.mdx

```diff
@@ -204,6 +204,17 @@ Release date: `November 4, 2025`
 
 ## Centreon Gorgone
 
+### 25.10.1
+
+Release date: `November 17, 2025`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Packaging] Fixed a issue that caused an upgrade failure due to centreon-perl-libs split into centreon-perl-libs and centreon-perl-libs-common.
```

Comment:
```suggestion
- [Packaging] Upgrade no longer fails because of centreon-perl-libs being split into centreon-perl-libs and centreon-perl-libs-common.
```